diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/comments/HeaderCommentRule.kt

```diff
@@ -190,6 +190,25 @@ class HeaderCommentRule(private val configRules: List<RulesConfig>) : Rule("head
         }
     }
 
+    private fun handleMultilineCopyright(copyrightText: String): String {
+        var properCopyright = ""
+        if (copyrightText.contains("\n")) {
+            copyrightText.split("\n").map { "    $it" }.forEach {
```

Comment:
Please check if this indent should be configured here or can be inside yml. E.g.,
```yml
copyrightText: |
      First line
      Second line
```
How many leading spaces does it have after deserialization?